e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -144,6 +144,8 @@ Scenario('HMCTS admin creates multiple orders for the case', async (I, caseViewP
   await createOrderEventPage.enterC21OrderDetails();
   await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
   await createOrderEventPage.enterJudgeAndLegalAdvisor(orders[0].judgeAndLegalAdvisor.judgeLastName, orders[0].judgeAndLegalAdvisor.legalAdvisorName);
+  await I.retryUntilExists(() => I.click('Continue'), '#orderFurtherDirections_directionsNeeded');
```

Comment:
as mentioned above, judiciary have permission for Create an order event so need to update judiciaryAdministersCase.. e2e test as well with same changes